singer/src/main/java/com/pinterest/singer/writer/kafka/CommittableKafkaWriter.java

```diff
@@ -295,9 +295,6 @@ public void endCommit(int numLogMessages, boolean isDraining) throws LogStreamWr
     try {
       writerFuture.get();
     } catch (CompletionException | InterruptedException | ExecutionException e) {
-      if (e instanceof InterruptedException) {
```

Comment:
Is this needed?

Reply:
It seems that you added this a while back https://github.com/pinterest/singer/commit/7d867c90c65e9f30b8818d3c026fc3d09b6ef669 but it wasn't added to the internal fork, do you recall if its needed?